src/vit-testing/iapyx/src/lib.rs

```diff
@@ -1,14 +1,33 @@
+//! Library part of iapyx which serves as a core lib for this testing CLI wallet.
+//! Idea behind split was to enable api interface for integration tests without spawning CLI process
+//! but directly operates on API level
+#![forbid(missing_docs)]
+#![warn(clippy::pedantic)]
+#![allow(
```

Comment:
Why we are omitting this clippy options for the whole crate ?
Maybe specify it for each case separately